pkg/openshift/templates/data-mysql.config_job.json.tpl.go

```diff
@@ -7,16 +7,17 @@ var DataMysqlConfigJob = `
   "apiVersion": "batch/v1",
   "kind": "Job",
   "metadata": {
-    "name": "{{index . "name"}}-mysql-dataconfig-job",
-	"labels": {
-		"rhmap/name":"datamysqlconfig"
-	}
+    "name": "{{if isset . "name"}}{{ index . "name"}}{{end}-mysql-dataconfig-job",
```

Comment:
Why would name be missing? This could cause problems as the name of the job needs to be unique, kubernetes wont launch a job if a job with the same name already exists which is quite likely with `-mysql-dataconfig-job` usually name comes populated with a hash to prevent job-name collisions.